src/openms/include/OpenMS/METADATA/SpectrumMetaDataLookup.h

```diff
@@ -283,6 +298,8 @@ namespace OpenMS
     /// Assignment operator (not implemented)
     SpectrumMetaDataLookup& operator=(const SpectrumMetaDataLookup&);
 
+    /// adds Ion matches to identifcations
+    static void addIonMatches_(PeptideIdentification& pi, const MSSpectrum<Peak1D> &spec);
```

Comment:
_re_ annotation?? I think this is not prevalent information in id. Also, it is lookup, as the data has to be gathered from the spectra data.